### PR TITLE
Minor edits to various topics

### PR DESCRIPTION
**Pull request by drothery-edb**
Updated 01_connect_cloud_account.mdx with heading change
Incorporated edits to Connecting to Your Cloud topic as discussed with Kelly

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [ ] This PR adds new content
- [ ] This PR changes existing content
- [ ] This PR removes existing content

### CODE REVIEW

File: product_docs/docs/edbcloud/beta/getting_started/01_connect_cloud_account.mdx [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: "Connecting Your Cloud Account"
 
 This topic describes how to set up your EDB Cloud subscription on Azure Marketplace. Your Azure subscription is where EDB Cloud can create and manage PostgreSQL clusters.
 
-## Before You Begin
+## Before You Vreate Your Subscription
```
Comment (drothery-edb) [inline product_docs/docs/edbcloud/beta/getting_started/01_connect_cloud_account.mdx:7]:
```suggestion
## Before You Create Your Subscription
```

File: product_docs/docs/edbcloud/beta/getting_started/01_connect_cloud_account.mdx [hunk 2]
```diff
@@ -43,27 +41,15 @@ To make requests to increase Azure Networking Limits follow [these instructions]
...
 
-3. Choose _application secret_ as an authentication option for the application. [Create a new Azure AD application secret. ](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#option-2-create-a-new-application-secret). After you created the secret, copy the value of Azure AD App Secret, you will need it for the final step.
+3. Choose _application secret_ as an authentication option for the application. [Create a new Azure AD application secret. ](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#option-2-create-a-new-application-secret) Take note of the Azure AD App Secret, you need it to configure your cloud account.
 
 4. If you can’t give away your existing Azure subscription, [create an additional Azure subscription](https://docs.microsoft.com/en-us/azure/cost-management-billing/manage/create-subscription).
```
Comment (kelpoole) [inline product_docs/docs/edbcloud/beta/getting_started/01_connect_cloud_account.mdx:50]: I don't understand what this statement means or why it is here

File: product_docs/docs/edbcloud/beta/getting_started/01_connect_cloud_account.mdx [hunk 3]
```diff
@@ -90,11 +75,15 @@ The following table describes the parameters that need to be entered in the EDB
 ![*Subscription progress](images/subscription-process.png "image_tooltip")
 
 4. Select **Configure account now** to configure EDB Cloud to use your Azure subscription and your Azure AD Application.
-5. Fill in the form using the values of parameters described in [Required Parameters](#Required-Parameters).
-   1. Azure AD App ID
-   2. Azure AD App Secret
-   3. Azure Subscription ID
-   4. Your EDB Cloud Organization Name
+5. Fill in the following parameters in the form: form using the values of parameters you noted when [creating your Azure AD Application](#create-azure-ad-application) and your EDB Cloud Origanization Name found on the overview page of a subscription.
```
Comment (kelpoole) [inline product_docs/docs/edbcloud/beta/getting_started/01_connect_cloud_account.mdx:78]: something seems off in this sentence with form: form

Comment (kelpoole) [inline product_docs/docs/edbcloud/beta/getting_started/01_connect_cloud_account.mdx:78]: in the section above this where is says to enter SaaS subscription name - that's in the subscription details section not the project details section - and we'll want to rever to that field name for Your EDB Cloud Organization I think

Comment (drothery-edb) [inline product_docs/docs/edbcloud/beta/getting_started/01_connect_cloud_account.mdx:78]:
```suggestion
5. Fill in the following parameters in the form.
```:

File: product_docs/docs/edbcloud/beta/getting_started/01_connect_cloud_account.mdx [hunk 4]
```diff
@@ -4,15 +4,13 @@ title: "Connecting Your Cloud Account"
 
 This topic describes how to set up your EDB Cloud subscription on Azure Marketplace. Your Azure subscription is where EDB Cloud can create and manage PostgreSQL clusters.
 
-## Before You Begin
+## Before You Create Your Subscription
```
Comment (aswright491) [inline product_docs/docs/edbcloud/beta/getting_started/01_connect_cloud_account.mdx:7]: I would leave this as "Begin" or something like "Prerequisites for Setting Up EDB Cloud". We aren't going to walk customers through how to create an Azure subscription.

File: product_docs/docs/edbcloud/beta/getting_started/01_connect_cloud_account.mdx [hunk 5]
```diff
@@ -4,15 +4,13 @@ title: "Connecting Your Cloud Account"
 
 This topic describes how to set up your EDB Cloud subscription on Azure Marketplace. Your Azure subscription is where EDB Cloud can create and manage PostgreSQL clusters.
```
Comment (aswright491) [inline product_docs/docs/edbcloud/beta/getting_started/01_connect_cloud_account.mdx:5]: I would change the first sentence to: "This topic describes how to set up your EDB Cloud account through the Microsoft Azure Marketplace."

File: product_docs/docs/edbcloud/beta/getting_started/01_connect_cloud_account.mdx (hunk 2 above)
Comment (aswright491) [reply product_docs/docs/edbcloud/beta/getting_started/01_connect_cloud_account.mdx:50]: The intent here is either if a user cannot use an existing subscription, then they will need to create a new subscription. Frankly, I would remove this altogether from here and state at the very beginning that a customer needs a Microsoft Azure subscription to get started.

**Review by aswright491 — CHANGES_REQUESTED** [review]
Left a few comments for changes

**Pull request merged**